src/app_service/service/settings/service.nim

```diff
@@ -508,6 +508,35 @@ QtObject:
       return true
     return false
 
+  proc showCommunityAssetWhenSendingTokens*(self: Service): bool =
+    return self.settings.showCommunityAssetWhenSendingTokens
+
+  proc toggleShowCommunityAssetWhenSendingTokens*(self: Service): bool =
+    let newValue = not self.settings.showCommunityAssetWhenSendingTokens
+    if(self.saveSetting(KEY_SHOW_COMMUNITY_ASSET_WHEN_SENDING_TOKENS, newValue)):
+      self.settings.showCommunityAssetWhenSendingTokens = newValue
+      return true
+    return false
+
+  proc displayAssetsBelowBalance*(self: Service): bool =
+    return self.settings.displayAssetsBelowBalance
+
+  proc toggleDisplayAssetsBelowBalance*(self: Service): bool =
+    let newValue = not self.settings.displayAssetsBelowBalance
+    if(self.saveSetting(KEY_DISPLAY_ASSETS_BELOW_BALANCE, newValue)):
+      self.settings.displayAssetsBelowBalance = newValue
+      return true
+    return false
+
+  proc displayAssetsBelowBalanceThreshold*(self: Service): int64 =
```

Comment:
Shouldn't this (and the respective QtProperty) be a float?

![image](https://github.com/status-im/status-desktop/assets/5377645/b7ac8881-dfd0-4159-b0bd-c53e64d27497)

Reply:
But the currency is coming from another user setting, not from status-go